src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/crud-table/crud-form/formly-array-type/formly-array-type.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="mb-3">
```

Comment:
Is it not a bit over-nested: `src/app/shared/datatable/crud-table/crud-form/formly-array-type/` ? What about putting this under `shared/crud-form`?

Reply:
yeah, locally I am moving this to `shared` now.

Reply:
Done. moved to `shared/form/crud-form`